engine/engine-mobile.xcodeproj/project.pbxproj

```diff
@@ -2066,7 +2066,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "cp \"$TARGET_BUILD_DIR/revsecurity.dylib\" \"$TARGET_BUILD_DIR/standalone-mobile-community.app/\"";
+			shellScript = "#cp \"$TARGET_BUILD_DIR/revsecurity.dylib\" \"$TARGET_BUILD_DIR/standalone-mobile-community.app/\"";
```

Comment:
Yes: otherwise, the building stops since revsecurity.dylib is not created anymore.
That can be better to remove the whole script though, as there is only this line.